demo/e2e-bdd/shared/pages/basePage.po.ts

```diff
@@ -0,0 +1,29 @@
+import { $, $$, browser, ElementArrayFinder, ElementFinder } from 'protractor';
+import { expect } from 'chai';
+
+export class BasePo {
+  headerLogo: ElementFinder = $('.logo');
+  headerIconBtns: ElementArrayFinder = $$('.header-list>li');
+  searchInput: ElementFinder = $('[name*="search"]');
+  bsVersionBtns: ElementArrayFinder = $$('.bootstrap-version .btn');
+  selectorForLeftNav = '.sidebar-list>li';
+  pageUrl: string;
```

Comment:
make this as get property and return full page url:  `(browser.baseUrl as string) + this.templateUrl;` instead of duplicating `(browser.baseUrl as string)` in each po